src/bigml/api/centroid.clj

```diff
@@ -23,10 +23,10 @@
   on failure unless :throw-exceptions is set to false (default is true),
   in which case the HTTP response details are returned as a map on failure."
   [cluster inputs & params]
-  (utils/create :target :centroid
-                :origin [:cluster cluster]
-                :params params
-                :inputs inputs))
+  (#'utils/create :target :centroid
```

Comment:
Using a reader macro to access a private fn is kind of unorthodox.  I think it's better to have `utils/create` as a public fn.